content/updates/2020-07/1-easily-fix-errors-on-your-connections.md

```diff
@@ -0,0 +1,15 @@
+# Easily fix errors on your connections 
```

Comment:
Don't forget the yaml at the beginning of the file to generate the PIM updates in the comm panel. You can find an example of the format in master (see old updates). :)

Reply:
Yes, I created the PR before our Tuesday meeting :) I'll add it at the beginning of the page